vendor/README.rst

```diff
@@ -0,0 +1,20 @@
+LLHTTP
```

Comment:
Can we try including this in the docs? Via `.. include::`.

Reply:
Could do, anywhere you have in mind to put it?

Reply:
OK, took a go at adding it after the dependencies list.

Reply:
@Dreamsorcerer I was kinda expecting this @ https://aiohttp--7367.org.readthedocs.build/en/7367/contributing.html. Not sure which one is better. llhttp could be mentioned in the deps, but the end-users wouldn't compile from Git anyway..